benchadapt/python/benchadapt/adapters/gbench.py

```diff
@@ -175,6 +174,7 @@ def _parse_results(self, results: dict, extra_tags: dict) -> List[BenchmarkResul
                 batch_id=batch_id,
                 extra_tags=extra_tags,
             )
+            result_parsed.optional_benchmark_info = {"gbench_context": gbench_context}
```

Comment:
Now or later: I think it makes sense to add a code comment here explaining what `gbench_context` is, or where one can read about it.

Reply:
Good idea; added it to the `_parse_gbench_json()` docstring, as that's where things are really happening.